### PR TITLE
post hour of code website changes

### PR DESCRIPTION
**Pull request by tanyaparker**
Changes
- [x] update code.org, csedweek.org, and hourofcode.com homepages
- [x] update og tags of all homepages to be evergreen
- [x] todo after merging: rescrape og tags and test on fb & twitter
- [x] remove prizes and volunteer section from hourofcode.com homepage
- [x] update HoC section on code.org homepage to include leaderboard instead of volunteers
- [x] remove prizes and signup from teacher banner at bottom of code.org/learn
- [x] teacher announcement on teacher dashboard to announce everything is back to normal
- [x] update volunteer pages to be year-round instead of hour of code

| code.org before | code.org after |
| --- | --- |

image
 | image

| hourofcode.com before | hourofcode.com after |
| --- | --- |
| image | image |

| csedweek.org before | csedweek.org after |
| --- | --- |
| image | image |

| code.org/volunteer before | code.org/volunteer after |
| --- | --- |
| image | image |

### CODE REVIEW

**Comment by tanyaparker** [comment]
@breville

**Comment by tanyaparker** [comment]
oops I removed the sw/mc 50/50 on the hoc block. adding back

**Comment by tanyaparker** [comment]
Nevermind, it's because my page mode is still the old 'mc'

File: pegasus/sites.v3/code.org/public/volunteer/index.haml [hunk 1]
```diff
@@ -1,20 +1,20 @@
 ---
-title: "Volunteer for the Hour of Code"
+title: "Volunteer"
```
Comment (caleybrock) [inline pegasus/sites.v3/code.org/public/volunteer/index.haml]: not super clear, especially for og tags. Maybe 'Volunteer in Computer Science Education'?

**Comment by caleybrock** [comment]
lgtm but I'd change page mode and test locally so frozen doesn't show up

**Comment by breville** [comment]
lgtm

**Pull request merged**